test/sourcemap.coffee

```diff
@@ -63,3 +64,55 @@ test "#3075: v3 source map fields", ->
   v3SourceMap = JSON.parse v3SourceMap
   arrayEq v3SourceMap.sources, ['tempus_fugit.coffee']
   eq v3SourceMap.sourceRoot, './www_root/coffee/'
+
+# Source maps aren't accurate on Node v12 ??
+if process.version.split(".")[0] != "v12"
+  test "native source maps", ->
+    new Promise (resolve, reject) ->
+      proc = spawn "node", [
```

Comment:
You can use `child_process.fork` to specifically spawn Node child processes: https://nodejs.org/api/child_process.html#:~:text=The%20child_process.fork()%20method%20is%20a%20special%20case%20of%20child_process.spawn()%20used%20specifically%20to%20spawn%20new%20Node.js%20processes.